salt/orch/update.sls

```diff
@@ -57,9 +57,10 @@ update_modules:
   salt.function:
     - tgt: {{ master_id }}
     - name: cmd.run
-    - bg: True
     - arg:
-      - systemctl reboot
+      - sleep 15; systemctl reboot
```

Comment:
Might be, the current code we are using does not use much magic: https://github.com/saltstack/salt/blob/2016.11/salt/modules/system.py#L95-L96

Maybe the problem was related to an immediate reboot as we have been experiencing running `systemctl reboot` ourselves.